numpy/core/src/multiarray/arrayobject.c

```diff
@@ -489,10 +501,15 @@ array_dealloc(PyArrayObject *self)
         }
         if (PyArray_FLAGS(self) & NPY_ARRAY_UPDATEIFCOPY) {
             /* DEPRECATED, remove once the flag is removed */
+            char const * msg = "ERROR: UPDATEIFCOPY detected in array_dealloc. "
+                " Required call to PyArray_ResolveWritebackIfCopy or "
+                "PyArray_DiscardWritebackIfCopy is missing";
```

Comment:
Not a fan of the word `ERROR` in a warning

Reply:
removed

Reply:
Remove the top one, but there is still an "ERROR" on line 500 to remove.

Reply:
fixed